Calculator.java

```diff
@@ -0,0 +1,12 @@
+class Calculator {
+
+	public static void main(String[] args){
+	System.out.println("hello world");
```

Comment:
Code is not intended correctly. Format code before checkin

Reply:
Hi sir, I have formatted the code now , pls check it